index.html

```diff
@@ -198,8 +198,6 @@ <h3>Motivation for a Profile</h3>
   <section id='sotd'></section>
 
   <section id="intro">
```

Comment:
Why do you want to remove a h1 heading?

Reply:
It's already commented out, this is just cleanup :)

Reply:
(Same with the previous two removals)

Reply:
https://github.com/w3c/wot-profile/pull/116